scheduler/scheduler.go

```diff
@@ -73,11 +52,37 @@ type Process struct {
 
 	// ulimit -u
 	Nproc uint
+}
+
+// Exposure controls the exposure settings for a process.
+type Exposure struct {
```

Comment:
This could also allow the consumer to specify the container port to use, but mostly unnecessary for now.